sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -392,4 +408,13 @@ public String getOctaneClassFactoryClassName() {
             return octaneClassFactoryClassName;
         }
     }
+
+    public static class OctaneCustomSettings {
+//        public static interface OctaneCustomSettings {
+//        public int readTimeout = 6000;
+//        public int connectionTimeout();
```

Comment:
do we need to leave these comments here ?

Reply:
definitely no

Reply:
Agree.  Please remove